pydantic/main.py

```diff
@@ -568,10 +568,10 @@ def parse_file(
         return cls.parse_obj(obj)
 
     @classmethod
-    def from_orm(cls: Type['Model'], obj: Any) -> 'Model':
+    def from_orm(cls: Type['Model'], obj: Any, **kwargs: Any) -> 'Model':
         if not cls.__config__.orm_mode:
             raise ConfigError('You must have the config attribute orm_mode=True to use from_orm')
-        obj = {ROOT_KEY: obj} if cls.__custom_root_type__ else cls._decompose_class(obj)
+        obj = {ROOT_KEY: obj} if cls.__custom_root_type__ else cls._decompose_class(obj, kwargs)
```

Comment:
Seems like, `kwargs` will be silently ignored if `cls.__custom_root_type__` is true. I don't think it should be the case.